consensus/parlia/parlia.go

```diff
@@ -2011,3 +2030,12 @@ func applyMessage(
 	}
 	return msg.Gas() - returnGas, err
 }
+
+// proposalKey build a key which is a combination of the block number and the proposer address.
+func proposalKey(header types.Header) string {
```

Comment:
double sign is not for sign at same block height, but sign on same block, e.g. with same ParentHash.
it should be: 
```
return header.ParentHash.String()+header.Coinbase.String()
```